### ISSUE TITLE
AttributeError: 'saas_portal.client' object has no attribute 'upgrade_db'

### ISSUE THREAD

**Issue by saltonmassally** [it-projects-llc/odoo-saas-tools#20]
@yelizariev  Clicking on Configure on clients throws - AttributeError: 'saas_portal.client' object has no attribute 'upgrade_db'... looking at saas_portal.client or saas_portal.database  shows no method called upgrade_db

**Comment by yelizariev** [comment]
Thanks. 
I've found related commit: https://github.com/yelizariev/odoo-saas-tools/commit/0bc39c807956bb561377e4fb0904d360a8880d82#diff-edd13e0aca827b70e6ba5c847c189a7c by @kaerdsar 
I've missed it when we made a lot of refactoring

upgrade_db should be restored at saas_portal.database
and upgrade_template should be replaced to 

```
@api.multi
def upgrade_template(self):
    return self[0].template_id.upgrade_db()
```

**Comment by kaerdsar** [comment]
Yes, I also missed the issue. I can fix it.

**Comment by kaerdsar** [comment]
Done!

**Comment by yelizariev** [comment]
fixed at #22
